src/validation_workflow/api_test.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+from typing import Any
+
+import requests
+
+apiHeaders_auth = {"Authorization": "Basic YWRtaW46YWRtaW4="}  # default user/pass "admin/admin" in Base64 format
+apiHeaders_accept = {"Accept": "*/*"}
+apiHeaders_content_type = {"Content-Type": "application/json"}
+apiHeaders = {}
+apiHeaders.update(apiHeaders_auth)
+apiHeaders.update(apiHeaders_accept)
+apiHeaders.update(apiHeaders_content_type)
```

Comment:
Please don't use globals.

Reply:
Thanks, @dblock , let me work on it and update!